docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+version: '3.9'
+services:
+  beyond:
+    build: .
+    tty: true
```

Comment:
What is the purpose of the tty key?

Reply:
It makes the container with shell, so that it could be detached successfully.
Otherwise, the container exits immediately after creation. This is happening by design when you dont run any application on it (right now we simply create the container without the app).